src/components/Button/Button.scss

```diff
@@ -14,8 +14,8 @@
     $button-background-color-focus: rgba($charcoal, .2) !default;
     $button-background-color-active: rgba($charcoal, .1) !default;
     $button-background-color-active-hover: rgba($charcoal, .05) !default;
-    $button-primary-background-top: #ff1966 !default;
-    $button-primary-background-bottom: #fa439f !default;
+    $button-primary-background-top: #951b81 !default;
+    $button-primary-background-bottom: #e6007e !default;
```

Comment:
Now that the gradient has been changed from top-to-bottom to left-to-right, please adjust the variable names accordingly.
I'd go with `-start` and `-end` to avoid future modifications if the gradient direction changes again.

Reply:
@jfrej Changed as suggested.